platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -665,6 +667,8 @@ public interface CoreI18n extends EmfI18n {
 		.de("Programm-UUID");
 	I18n0 PROGRAMS = new I18n0("Programs")//
 		.de("Programme");
+	I18n0 PROGRAMS_WILL_NOT_BE_EXECUTED = new I18n0("Programs will not be executed.")//
```

Comment:
rather `No new program executions will be launched.` and `Es werden keine neuen Programmausführungen gestartet.`